### ISSUE TITLE
Two graphs at the same time, one behind the other

### ISSUE THREAD

**Issue by Lupici** [amcharts/amcharts4#3912]
Hi all, I'd like a reply to my question, like "yes, amcharts allow to do this", or "no, amcharts does't allow to do this".....

I need to have 2 graphs in the same "plot" container. I try to explain

![image](https://user-images.githubusercontent.com/89850538/143859385-fd532967-096c-40fe-b558-83a680ab8731.png)

In the picture there is 2 bars on the left (that can become 3, 4...) with a gray background. These bars have to stay on the left, in a fixed position.

The other bars can scroll from left to right (with an horizonatal scrollbar) going behind the gray background. Also because I can have 50 bars, for example, and I can see 5 or 6 of them and the other remain hidden (to the left or to the right), and I can use the scrollbar to see the others.

Basically this is the effect that I can try to achieve. Does amcharts allow this? and if yes, I can achieve it?

Thanks a lot

L.

**Comment by martynasma** [comment]
No, I'm afraid there's no way to do what you have described. Sorry.
